### PR TITLE
add brubsby.com presence

### PR DESCRIPTION
**Pull request by brubsby**
I have a personal website that hosts a random media player related to discord servers that I'm in, so I'd like to add this presence that tells people what media the user is viewing (and that my website is being used).

thanks,
brubsby

### CODE REVIEW

File: websites/B/brubsby/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    "id": "163145773964328960"
+  },
+  "url": "brubsby.com",
+  "description": {
+    "en": "english"
+  },
+  "service": "brubsby.com",
+  "version": "1.0.0",
+  "logo": "https://brubsby.com/assets/favicon.png",
+  "thumbnail": "https://brubsby.com/assets/favicon.png",
```
Comment (Bas950) [inline websites/B/brubsby/dist/metadata.json:14]: upload to imgur

File: websites/B/brubsby/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.0",
+  "author": {
+    "name": "brubsby",
+    "id": "163145773964328960"
+  },
+  "url": "brubsby.com",
+  "description": {
+    "en": "english"
+  },
```
Comment (Bas950) [inline websites/B/brubsby/dist/metadata.json:10]: English is not a description

File: websites/B/brubsby/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  "service": "brubsby.com",
+  "version": "1.0.0",
```
Comment (Bas950) [inline websites/B/brubsby/dist/metadata.json:12]: service name and folder name must be the same

File: websites/B/brubsby/presence.ts [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+    }
+  }
+
+  if (presenceData.details == null) {
+    //This will fire if you do not set presence details
+    presence.setTrayTitle(); //Clears the tray title for mac users
+    presence.setActivity(); /*Update the presence with no data, therefore clearing it and making the large image the Discord Application icon, and the text the Discord Application name*/
+  } else {
+    //This will fire if you set presence details
+    presence.setActivity(presenceData); //Update the presence with all the values from the presenceData object
```
Comment (Bas950) [inline websites/B/brubsby/presence.ts:59]: remove comments

**Review by Bas950 — CHANGES_REQUESTED** [review]
Upload proof of it working

**Comment by brubsby** [comment]
> Upload proof of it working

image

File: websites/B/brubsby.com/dist/metadata.json [hunk 5]
```diff
@@ -6,13 +6,13 @@
...
   "url": "brubsby.com",
   "description": {
-    "en": "english"
+    "en": "brubsby.com hosts the random siivagunner rip player, pen plotter tool cozyvec, and other personal projects from brubsby"
   },
   "service": "brubsby.com",
   "version": "1.0.0",
-  "logo": "https://brubsby.com/assets/favicon.png",
-  "thumbnail": "https://brubsby.com/assets/favicon.png",
+  "logo": "https://imgur.com/Miem17T.png",
```
Comment (Bas950) [inline websites/B/brubsby.com/dist/metadata.json]: thats not really a logo....

File: websites/B/brubsby.com/dist/metadata.json [hunk 6]
```diff
@@ -6,13 +6,13 @@
...
+  "thumbnail": "https://imgur.com/Miem17T.png",
   "color": "#FFFFFF",
```
Comment (Bas950) [inline websites/B/brubsby.com/dist/metadata.json]: thumbnail most be a promotional banner or a screenshot of the website homepage.

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Comment by ririxidev** [comment]
no response from author - closing.

**Pull request closed without merging**